pymatgen/electronic_structure/cohp.py

```diff
@@ -971,40 +971,24 @@ def __init__(
     def __str__(self) -> str:
```

Comment:
@JaGeo @QuantumChemist  I simplifies this method a little bit, and I don't expect any change in results here. But still good to ask for your review in case I missed anything. Thanks a lot :)

Reply:
Tagging @naik-aakash as he has likely written parts of it originally.

Reply:
Sure. Thanks for the input :)

Reply:
Looks good but I also think Aakash can tell you more.

Reply:
Thanks! I guess he missed this one because we pinged him twice @naik-aakash :)

Reply:
Hi @DanielYang59, you are right! Somehow, I missed this one. Really sorry. The changes seem fine to me as well. 😄

Reply:
> Somehow, I missed this one. Really sorry. 

That's because we pinged you in the same PR twice, certainly the notification got overwritten :) All good.

> The changes seem fine to me as well. 😄

Great! thanks for the confirmation :)